lib/maxmind/db/file_reader.rb

```diff
@@ -0,0 +1,43 @@
+require 'maxmind/db/errors'
+
+module MaxMind # :nodoc:
+  class DB
+    class FileReader # :nodoc:
+      def initialize(filename)
+        @fh = File.new(filename, 'rb'.freeze)
+        @size = @fh.size
+        @mutex = Mutex.new
+      end
+
+      def size
+        @size
+      end
+
+      def close
+        @fh.close
+      end
+
+      def read(offset, size)
+        if size == 0
+          return "".freeze.b
```

Comment:
I think you're adopted Option (a) from here: https://github.com/rubocop-hq/ruby-style-guide#strings

which would mean use `''` instead of `""`.

Reply:
Good catch.